File: VINocr/vinOCR.py
import xml.etree.ElementTree as ET

import requests

from PIL import Image
import os

link = "http://www.recognition.ws/vinocr/v1?accesscode=7fdc7ba0-6a5c-4122-a74c-31f5de9ef87c"
# YOUR_ACCESS_CODE: Your access code.
path = "C:/Users/sooph/Downloads/2751-0ab4bd62-d696-4fb5-83d8-3eee708732fc.jpg"


image = open(path, 'rb')

with open(path, 'rb') as img:
  imgName = os.path.basename(path)
  files = {'image': (imgName, img)}
  with requests.Session() as s:
    r = s.post(link, files = files)
    print(r.text)
    xml = ET.fromstring(r.text)
    output = "Captured VIN: " + xml[0].text
    print(output)

File: VINocr/vinOCR.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net;
using System.Xml.Linq;
using System.Drawing;

namespace VINocr
{
    class Program
    {
        static void Main(string[] args)
        {
            String url = "http://www.recognition.ws/vinocr/v1?accesscode=YOUR_ACCESS_CODE";
            // YOUR_ACCESS_CODE: Your access code.

            String path = "YOUR_FILE_PATH";
            // YOUR_FILE_PATH: Your file path.

            WebClient client = new WebClient();
            Image barcode = Image.FromFile(path);
            byte[] response = client.UploadFile(url, path);
            String responsestring = client.Encoding.GetString(response);
            XElement root = XElement.Parse(responsestring);

            // Iterating for "VIN_Captured" item in the root tag.

            var el = root.Elements("VIN_Captured");
            Console.WriteLine("VIN Captured: " + (string) el.Value);

        }
    }
}
